src/Mapster/Interfaces/IMapFrom.cs

```diff
@@ -2,8 +2,12 @@ namespace Mapster;
 
 public interface IMapFrom<TSource>
 {
-    //public void ConfigureMapping(TypeAdapterConfig config)
-    //{
-    //    config.NewConfig(typeof(TSource), GetType());
-    //}
+
```

Comment:
This fix test, but not fix problem. 
Temporarily target mapster.Test to net48 and you see it.
IMapFrom  looks like for a netstandard 2.0  should be a different implementation or completely cut out.

Reply:
Ok, I will have to try that next. Do you have a suggestion for fixing it?

Reply:
I figured out how to cut it out. But there is no way to make it work yet.
https://github.com/DocSvartz/Mapster/tree/net-standart-2.0
Last two commits to Related to this issue.

Reply:
Is it necessary to have it target net48? When it previously targeted net6.0/netcor3.1 wasn't this all working in .net framework pre Mapster 7.4?

Reply:
Before version 7.4.0 Mapster was target at netstandard 2.0, 1.3. those. The development itself ensured that features that are available in newer TFM would not be explicitly used.

Compatibility is always backwards. Net 6.0 can test everything that came before it. Because he himself supports it.
Here is a different case.
To make it work you had to look for workarounds  you build in and disable features that are not included in the target platform.
Therefore, I think that it is necessary to test that the behavior of the api does not differ for different target platforms.

My answer is, no, not necessarily. But then there is no guarantee that everything will work as intended in an environment that really only supports netstandard 2.0.

Reply:
ImapFrom.cs
```
#if NETSTANDARD2_1_OR_GREATER
public interface IMapFrom<TSource>
{

    public void ConfigureMapping(TypeAdapterConfig config)
    {
        config.NewConfig(typeof(TSource), GetType());
    }
    
}
#endif

#if !NETSTANDARD2_1_OR_GREATER
public interface IMapFrom<TSource>
{
    public void ConfigureMapping(TypeAdapterConfig config);
    
}
public static class MapFromLegacyHelper 
{
    public static void MapFromConfigureDefault<TSource>(this IMapFrom<TSource> mapFrom, TypeAdapterConfig config)
    {
        config.NewConfig(typeof(TSource), mapFrom.GetType());
    }

}

#endif
```

From Test where need implementation

```
#if !NETSTANDARD2_1_OR_GREATER
    public void ConfigureMapping(TypeAdapterConfig config)
    {
        this.MapFromConfigureDefault(config);
    }
#endif
```

Reply:
It looks like it's just a registration helper.
In tests, even if you don’t specify an implementation, it will still work.
```

public void ConfigureMapping(TypeAdapterConfig config)
{
}
```